tests/behat/features/unsaved-changes-indicator.feature

```diff
@@ -0,0 +1,45 @@
+@javascript @retry @job2
+Feature: Unsaved changes indicator
+  As an author
+  I want to be alerted of unsaved changes
+
+  Background:
+    # Config will put ~2 second delay until the unsaved changes notice shows
+    # Note - intentionally not testing the warning time because there's likely to
+    # be flakiness with the behat test due to timing issue e.g. CI infra being slow
```

Comment:
Why don't we just wait a couple of seconds longer than we expect to need to?

The likelyhood of flakiness with notice->warning is presumably the same likelyhood of flakiness for dirty->notice so I don't see why we would test one and not the other?

Reply:
We already have comprehensive testing using react-testing-library for the time based state updates so shouldn't duplicate in behat which would only make things run slower. I've updated the comment to note this